modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,198 @@
+
+* This update adds a new field named `default-resolver-type` in the `default-configs` config map. You can set the value of this field to configure a default resolver.
+
+* With this update, you can define and use a `PipelineRun` context variable in the `pipelineRun.workspaces.subPath` configuration.
+
+
+[id="triggers-new-features-1-11_{context}"]
+=== Triggers
+
+* With this update, Tekton Triggers support the `Affinity` and `TopologySpreadConstraints` values in the `EventListener` specification. You can use these values to configure Kubernetes and custom resources for an `EventListener` object.
+* This update adds a Slack interceptor that allows you to extract fields by using a slash command in Slack. The extracted fields are sent in the form data section of an HTTP request.
+
+
+[id="operator-new-features-1-11_{context}"]
+=== Operator
+
+* With this update, you can configure pruning for each pipeline run or task run resource by setting a boolean field named `prune-per-resource` in the `TektonConfig` CR. You can also configure pruning for each pipeline run or task run resource in a  namespace by adding the annotation, `operator.tekton.dev/prune.prune-per-resource=true`, to that namespace.
+* With this update, if there are any changes in the {product-title} cluster-wide proxy, Operator Lifecycle Manager (OLM) recreates the {pipelines-title} Operator. 
+* With this update, you can disable the pruner feature by setting the `config.pruner.disabled` field to `true` in the `TektonConfig` CR.
+* This update increments the version number of pipelines to 0.47.
+* This update increments the version number of {pac} to 0.19.
+
+
+[id="resolver-new-features-1-11_{context}"]
+=== Resolvers
+
+* With this update, the `ClusterResolver` feature is now available by default. You can use this feature to fetch tasks and pipelines from a configured namespace. 
+* With this update, the `BundleResolver` feature is now available by default. You can use this feature to fetch tasks and pipelines from a Tekton bundle. 
++
+[NOTE]
+====
+You must prefer the `BundleResolver` feature over the `taskref.bundle` or `pipelineref.bundle` syntax.
+====
+
+[id="chains-new-features-1-11_{context}"]
+=== {tekton-chains}
+
+* With this update, {tekton-chains} is now generally available for use.
+
+* With this update, you can use the skopeo tool with {tekton-chains} to generate keys, which are used in the `cosign` signing scheme. 
+
+
+[id="tekton-hub-new-features-1-11_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* This update adds a new endpoint `resource/<catalog_name>/<kind>/<resource_name>/raw` and a new field named `resourceURLPath` in the `resource` API response. This update helps you to obtain the latest raw YAML file of the resource.
+
+[id="tekton-results-new-features-1-11_{context}"]
+=== Tekton Results
```

Comment:
Can we write Tech Preview here? Tekton Results (Tech Preview)

Reply:
I have already added a TP note, you can view it in the preview: https://59661--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#new-features-1-11_op-release-notes